setup/product_brand_stock/odoo/addons/product_brand_stock

```diff
@@ -0,0 +1 @@
+../../../../product_brand_stock
```

Comment:
This shouldn't be here in the migration commit but in the precommit commit.

BTW, since title is not only for autoformatting but also for things like these ones, title should be:

`[IMP] module_name: pre-commit stuff`

(according to OCA guidelines).

Reply:
`pre-commit-stuff` -> `pre-commit stuff` (remove extra "-").